pkg/run/state.go

```diff
@@ -5,6 +5,7 @@ import (
 	"sync"
 )
 
+// State keeps track of the current stats
 type State struct {
```

Comment:
Should this struct and its methods be public?

Reply:
That's an interesting question :+1: 
I planned to move it in a dedicated package as you might observe in #57